<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Youka_User_Window</class>
 <widget class="QMainWindow" name="Youka_User_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>优卡用户查询</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="table_results">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>711</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>671</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item row="0" column="1" rowspan="2">
      <widget class="QDateTimeEdit" name="date_from"/>
     </item>
     <item row="1" column="2" rowspan="2">
      <widget class="QPushButton" name="btn_Query">
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3" rowspan="2">
      <widget class="QPushButton" name="btn_ExportCsv">
       <property name="text">
        <string>导出</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbl_EndTime">
       <property name="text">
        <string>结束时间：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDateTimeEdit" name="date_end"/>
     </item>
     <item row="0" column="0" rowspan="2">
      <widget class="QLabel" name="lbl_StartTime">
       <property name="text">
        <string>起始时间：</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
